src/Model/ValueObject/Identity/UUID.php

```diff
@@ -27,4 +27,9 @@ public function __construct(string $value)
 
         $this->setValue($value);
     }
+
+    public static function uuid4(): self
```

Comment:
I think it's tricky to place that here, look at the comment inside the constructor. It also supports another legacy format.

Also if you would like to add this, please add it in another pull request and modify all existing `Uuid::uuid4()` to `UUID::uuid4()` because you did not take into account that PHP ignores casing.